src/gossip/service.zig

```diff
@@ -2920,6 +2938,7 @@ test "test large push messages" {
     defer {
         gossip_service.shutdown();
         gossip_service.deinit();
+        allocator.destroy(gossip_service);
```

Comment:
Is there an issue with calling self.allocator.destroy from within the deinit method?

Reply:
if we did we would make the assumption that the service is always on the heap which i dont think we want to make